### DISCUSSION TITLE
v4.18.1

### DISCUSSION THREAD

**Discussion by dhui** [golang-migrate/migrate#1155] [Announcements]
## Changes
* Update dktest from v0.4.2 to v0.4.3

**Full Changelog**: https://github.com/golang-migrate/migrate/compare/v4.18.0...v4.18.1

<hr /><em>This discussion was created from the release <a href='https://github.com/golang-migrate/migrate/releases/tag/v4.18.1'>v4.18.1</a>.</em>

**Comment by emetsger** [comment]
Is there a reason the darwin arm binary doesn't have sqlite support?

**Reply by dhui** [reply]
https://github.com/golang-migrate/migrate/issues/59
https://github.com/golang-migrate/migrate/pull/555#pullrequestreview-649665127
